match2/commons/match_group_display_bracket.lua

```diff
@@ -610,7 +599,7 @@ function BracketDisplay.NodeLowerConnectors(props)
 
 	-- Compute partial sums of heights of lower round matches
 	local heightSums = Math.partialSums(
-		Array.map(lowerLayouts, function(lyt) return lyt.height end)
+		Array.map(lowerLayouts, function(layout_) return layout_.height end)
```

Comment:
```suggestion
		Array.map(lowerLayouts, function(l) return l.height end)
```

Reply:
I think using `layout_` is fine.

Reply:
I don't, otherwise I wouldn't comment it

Reply:
ok, changed it